### ISSUE TITLE
No web app is loaded http//0.0.0.0:5000/

### ISSUE THREAD

**Issue by erichello** [defaultnamehere/zzzzz#32]
_
Sorry, I've never written code or anything, I literally have no programming experience and this is the first creation that I decided to attempt to use. 
_

I ran python graph.py
then python app.py
then used chrome and mozilla firefox to access both 0.0.0.0:5000 and localhost:8000; got the "This webpage is unavailable" error for both.

What did I do wrong?

I think I got through all the steps (unless SECRET.txt has to be formatted in some different way?); all the information is right and cmd is no longer giving me an error.

**Comment by pielco11** [comment]
Can you post the output, please?

**Comment by infinia-yzl** [comment]
Having the same problem. What output do you need?

**Comment by pielco11** [comment]
Well, if you are sure that fetcher.py and graph.py work correctly you can not post them. So can you post the output of app.py, please?

**Comment by erichello** [comment]
Restarting with stat
Debugger is active!
Debugger pin code: 473 - 957 - 672
Running on http://0.0.0.0:5000/ (Press Ctrl+C to quit)

**Comment by pielco11** [comment]
Well, I see no errors. So are you sure about SECRETS.txt? When you execute fetcher.py, do you see some informations or something else?

**Comment by erichello** [comment]
I'm not exactly sure about SECRETS.txt;
I found and copied the clientid, uid, and cookie from 10 digit code using copy cURL (cmd) on facebook.com/pull.. etc, where they were:
1. client-id= ___ 
2. cookie = 'datr=_____; lu=______; ally=______; c_user=______; fr=____;xs=____; csm=_______, s= _______, p=________, act=________; presence=______'
   (I didn't put in the --compressed part)
3. uid=____ 

Is this wrong? 

I see the csv files and I can open them with the data in it.

**Comment by pielco11** [comment]
Well, it is like mine. And mine works, so at this point I don't know how to help you.

**Comment by erichello** [comment]
Got it; found it on localhost:5000

**Comment by ParvJain** [comment]
#33 will fix it.

**Comment by ghost** [comment]
Instead of going to 0.0.0.0:5000, try 127.0.0.1:5000 or localhost:5000

**Comment by alekz7** [comment]
In my case another application was ussing the port 5000 so I changed in my code to port 5040 and try localhost:5040, works for me!

**Comment by amiralizade2** [comment]
in program.cs
change: 
if (isDevelopment)
          webHost = webHost.UseKestrel().UseUrls("http://0.0.0.0:5000");

TO:
if (isDevelopment)
          webHost = webHost.UseKestrel().UseUrls("http://127.0.0.1:5000");

**Comment by Daniel15** [comment]
> Instead of going to 0.0.0.0:5000, try 127.0.0.1:5000 or localhost:5000

Yeah `0.0.0.0` isn't a routable address. When something says "`Running on http://0.0.0.0:5000`", it means that it's listening to port 5000 on all IPs, and you should actually use a real IP or hostname to access it, not literally go to `0.0.0.0:5000` in your browser.

**Comment by vijayakumarpattanashetti** [comment]
I too faced similar issue, was able to fix it by replacing:

app.run(debug=False)

with

app.run(host='0.0.0.0', debug=False)


If multiple hosts, then host = list(str(<hosts>))
